### ISSUE TITLE
Implement fight map modifiers

### ISSUE THREAD

**Issue by Zenoo** [Zenoo/labrute#748]
- List all modifiers to implement
- Link modifiers to their fight backgrounds
- Find free to use artwork for modifiers that don't fit with any existing background
- Add odds for each fight background, depending on the modifiers linked
- Use these odds in a weighted random roll instead of pure random when generating a fight
- Display the map modifiers in play when viewing a fight

**Comment by MatejNota** [comment]
Hello, can I work in this issue?

**Comment by Zenoo** [comment]
Everyone can work on whatever issue they want to @MatejNota, but this one requires a lot of research beforehand, just a heads up.
I updated the description with the steps to follow.

**Comment by MatejNota** [comment]
OK so I have some questions:
1. What should the modifiers modify? Maybe something about brute stats? Or should they add some status effects?
2. Should every backround have some effect? Or should I create new "fight backrounds"?
3. Are these modifiers only for fights between brutes? Or does it apply to boss fights also?

**Comment by MatejNota** [comment]
I found these modifiers, are those the ones I should link to backrounds?
```

export const FightModifier: {
  noThrows: 'noThrows',
  focusOpponent: 'focusOpponent',
  alwaysUseSupers: 'alwaysUseSupers',
  drawEveryWeapon: 'drawEveryWeapon',
  doubleAgility: 'doubleAgility',
  randomSkill: 'randomSkill',
  randomWeapon: 'randomWeapon',
  bareHandsFirstHit: 'bareHandsFirstHit',
  startWithWeapon: 'startWithWeapon'
};
```

**Comment by Zenoo** [comment]
No, these modifiers are those used for the daily modifiers that pop up from time to time.
They are not linked to maps.

1. The modifiers have to be created from scratch, and have to be logical for specific maps. For example, for a swampy map, reduce the speed by 50%, etc etc ...
2. Some backgrounds should have modifiers, yes, whether they are existing or new ones. The main one(s) should not have any modifiers, so that getting one is rare.
3. They apply on any fight

**Comment by MatejNota** [comment]
I did some research, and I am starting to work on implementation (for now only backend, no visuals). For now I will try to enumerate mapModifiers in prisma and see how will it affects fights.
